README.md

```diff
@@ -70,9 +70,9 @@ sudo ./scripts/stop
 - [`Umbrel Dashboard`](https://github.com/getumbrel/umbrel-dashboard)
 - [`Umbrel Manager`](https://github.com/getumbrel/umbrel-manager)
 - [`Umbrel Middleware`](https://github.com/getumbrel/umbrel-middleware)
-- [`Bitcoin Core`](https://github.com/getumbrel/docker-bitcoind)
-- [`LND`](https://github.com/getumbrel/docker-lnd)
-- [`Tor`](https://github.com/getumbrel/docker-tor)
+- [`Bitcoin Core`](https://github.com/lncm/docker-bitcoind)
+- [`LND`](https://github.com/lncm/docker-lnd)
+- [`Tor`](https://github.com/lncm/docker-tor)
 - [`Nginx`](https://github.com/nginx/nginx)
 - [`Neutrino Switcher`](https://github.com/lncm/docker-lnd-neutrino-switch)
```

Comment:
I think this one should point to the modified Unmbrel repo, I'm not sure if this should be part of this PR

Reply:
Modified?

Reply:
With modified I meant that it's not the same as your repo, there are a few differences https://github.com/getumbrel/docker-lnd-neutrino-switch

Reply:
I didn't change that..

Reply:
Yes, but I just wanted to make @lukechilds and @mayankchhabra aware of that, and you maybe could add changing the URL to Umbrels container to that PR too.

Reply:
they are using their own fork of it, the doc pr just points the dependencies correctly so that anyone who finds issues with bitcoind / lnd / tor knows where to look.